app/main/lib/reindex_images.py

```diff
@@ -0,0 +1,46 @@
+from flask import current_app as app
+from flask import request, current_app as app
+from flask import current_app as app
+from app.main import db
+from app.main.model.image import ImageModel
+from app.main.lib.helpers import context_matches
+from app.main.lib.similarity_helpers import get_context_query, drop_context_from_record
+from app.main.lib.image_similarity import add_image_pdq
+
+from sqlalchemy import text
+from sqlalchemy.orm.attributes import flag_modified
+from sqlalchemy.orm.exc import NoResultFound
+
+app.app_context()
+def _after_log(retry_state):
+  app.logger.debug("Retrying image similarity...")
+
+def get_all_images():
```

Comment:
Only get images `WHERE pdq IS NULL` . Add a `LIMIT` parameter or we'll exhaust RAM